assets/dev/js/frontend/handlers/base-nested-tabs.js

```diff
@@ -154,10 +154,20 @@ export default class BaseNestedTabs extends Base {
 	activateTab( tabIndex ) {
 		const settings = this.getSettings(),
 			activeClass = settings.classes.active,
-			$requestedTitle = this.elements.$tabTitles.filter( this.getTabTitleFilterSelector( tabIndex ) ),
-			$requestedContent = this.elements.$tabContents.filter( this.getTabContentFilterSelector( tabIndex ) ),
 			animationDuration = 'show' === settings.showTabFn ? 0 : 400;
 
+		let $requestedTitle = this.elements.$tabTitles.filter( this.getTabTitleFilterSelector( tabIndex ) ),
```

Comment:
consider using .find instead of .filter

Reply:
@TzviRabinovitch 
It doesn't seem to work for me. 
But one clarification: we only use the alternative values in case we remove the latest tab if it is the active tab. In other words. In 99.9% of the cases we will use the first declared `$requestedTitle` and `$requestedContent` values.